src/components/shared/Container.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Container = ({children}: {children: React.ReactNode}) => {
+  return (
+    <div className="mx-auto w-full max-w-[767px] px-[16px] tablet:max-w-[1279px] desktop:max-w-[1440px] desktop:px-[24px]">
```

Comment:
While it's not an issue, consider using Tailwind shorthand values (**p-4** for **px-[16px]**, **max-w-screen-lg** instead of exact pixel values) where possible, as they improve readability and maintainability.

Reply:
I halfway agree. Padding widths can be adjusted, but the maximum widths used in our project do not match the default width values in tailwind styles.